sbom-utility-scripts/scripts/base-images-sbom-script/app/test_base_images_sbom_script.py

```diff
@@ -516,6 +517,131 @@ def test_main_input_sbom_does_not_contain_formulation(tmp_path, mocker):
+                "externalRefs": [
+                    {
+                        "referenceType": "purl",
+                        "referenceLocator": "pkg:oci/single-container-app@sha256"
+                        ":8f99627e843e931846855c5d899901bf093f5093e613a92745696a26b5420941?repository_url"
+                        "=quay.io/mkosiarc_rhtap/single-container-app",
+                        "referenceCategory": "PACKAGE-MANAGER",
+                    }
+                ],
+                "annotations": [
+                    {
+                        "annotator": "Tool:konflux:jsonencoded",
+                        "annotationDate": "2021-07-01T00:00:00Z",
+                        "annotationType": "OTHER",
+                        "comment": '{"name":"konflux:container:is_builder_image:for_stage","value":"0"}',
+                    }
+                ],
+            },
+            {
+                "name": "registry.access.redhat.com/ubi8/ubi",
+                "SPDXID": "SPDXRef-container-registry.access.redhat.com/ubi8/ubi-"
+                "0f22256f634f8205fbd9c438c387ccf2d4859250e04104571c93fdb89a62bae1",
+                "downloadLocation": "NOASSERTION",
+                "externalRefs": [
+                    {
+                        "referenceCategory": "PACKAGE-MANAGER",
+                        "referenceType": "purl",
+                        "referenceLocator": "pkg:oci/ubi@sha256:"
+                        "627867e53ad6846afba2dfbf5cef1d54c868a9025633ef0afd546278d4654eac"
+                        "?repository_url=registry.access.redhat.com/ubi8/ubi",
+                    }
+                ],
+                "annotations": [
+                    {
+                        "annotator": "Tool:konflux:jsonencoded",
+                        "annotationDate": "2021-07-01T00:00:00Z",
+                        "annotationType": "OTHER",
+                        "comment": '{"name":"konflux:container:is_base_image","value":"true"}',
+                    }
+                ],
+            },
+        ],
+        "relationships": [
+            {
+                "relatedSpdxElement": "SPDXRef-DocumentRoot-Unknown-",
+                "relationshipType": "DESCRIBES",
+                "spdxElementId": "SPDXRef-Document",
+            },
+            {
+                "relatedSpdxElement": "SPDXRef-DocumentRoot-Unknown-",
+                "relationshipType": "BUILD_TOOL_OF",
+                "spdxElementId": "SPDXRef-container-quay.io/mkosiarc_rhtap/"
+                "single-container-app-9520a72cbb69edfca5cac88ea2a9e0e09142ec934952b9420d686e77765f002c",
+            },
+            {
+                "spdxElementId": "SPDXRef-container-registry.access.redhat.com/"
+                "ubi8/ubi-0f22256f634f8205fbd9c438c387ccf2d4859250e04104571c93fdb89a62bae1",
+                "relationshipType": "BUILD_TOOL_OF",
+                "relatedSpdxElement": "SPDXRef-DocumentRoot-Unknown-",
+            },
```

Comment:
Could you keep `spdxElementId` and `relatedSpdxElement` in a consistent order, ideally `spdxElementId` first?

At first, I thought "Wait, how the heck does `SPDXRef-DocumentRoot-Unknown-` describe the `SPDXRef-Document`" until I noticed the order was inconsistent